content/en/real_user_monitoring/browser/advanced_configuration.md

```diff
@@ -19,6 +19,73 @@ further_reading:
 
 Find below the different initialization options available with the [Datadog Browser SDK][1].
 
+### Scrub sensitive data from your RUM data
+If your RUM data contains sensitive information that need redacting, configure the Browser SDK to scrub sensitive sequences by using the `beforeSend` callback when you initialize RUM.
+
+This callback function gives you access to every event collected by the RUM SDK before they get sent to Datadog. 
+
+For example, redact email addresses from your web application URLs:
+
+{{< tabs >}}
+{{% tab "NPM" %}}
+
+```javascript
+import { Datacenter, datadogRum } from '@datadog/browser-rum';
+
+datadogRum.init({
+    ...,
+    beforeSend: (event) => {
+        // remove email from view url
+        event.view.url = event.view.url.replace(/\S+@\S+\.\S+/, "REDACTED_EMAIL")
```

Comment:
your regex [seems a bit greedy](https://regex101.com/r/YlWdCm/2)
it could be nice to provide a working example

Reply:
Argh, I'll stick to the example you provided on the PR description then :D That's what happens when I try to be smart!